front/app/api/authentication/sign_in_out/signOut.ts

```diff
@@ -33,10 +31,6 @@ export default async function signOut() {
       ) {
         clHistory.push('/');
       }
-
-      if (pathname && endsWith(pathname, '/ideas/new')) {
-        clHistory.push(pathname.split('/ideas/new')[0] as RouteType);
-      }
```

Comment:
This redirected us back to the project on signing out. Now, however, we show the Unauthorized page so the need for this doesn't exist anymore.